profile-widget/diveprofileitem.cpp

```diff
@@ -421,12 +421,15 @@ void DivePercentageItem::paint(QPainter *painter, const QStyleOptionGraphicsItem
 	QPolygonF poly = polygon();
 	for (int i = 1, modelDataCount = dataModel->rowCount(); i < modelDataCount; i++) {
 		if (i < poly.count()) {
-			double value = dataModel->index(i, vDataColumn).data().toDouble();
-			int cyl = dataModel->index(i, DivePlotDataModel::CYLINDERINDEX).data().toInt();
-			int inert = 1000 - get_o2(&displayed_dive.cylinder[cyl].gasmix);
-			mypen.setBrush(QBrush(ColorScale(value, inert)));
-			painter->setPen(mypen);
-			painter->drawLine(poly[i - 1], poly[i]);
+			double value = dataModel->index(i - 1, vDataColumn).data().toDouble();
+			int sec = dataModel->index(i - 1, hDataColumn).data().toInt();
```

Comment:
why are we shifting to i-1 as index? I'm sure that this will become obvious once I stare more at the bigger context of the code, but I would love to see the explanation for that in the commit message